src/components/body/Body.tsx

```diff
@@ -73,7 +77,6 @@ function isTargetWithinScrollArea(
   const bodyHeight = scrollRoot.clientHeight;
 
   return (
-    scrollRoot.scrollTop + bodyHeight - (target as HTMLElement)?.offsetTop <
-    bodyHeight
+    scrollRoot.scrollTop + bodyHeight - target?.offsetTop < bodyHeight
```

Comment:
This calculation doesn't apply here any longer. When you face the conflict rebasing, just omit it.